diff -r 993591e7a9ee -r 3d6f83ca9716 awia/=BTC=_TK_punishment/=BTC=_punish_action.sqf
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/awia/=BTC=_TK_punishment/=BTC=_punish_action.sqf	Fri Sep 27 15:20:23 2013 +0400
@@ -0,0 +1,7 @@
+_id = _this select 2;
+_array = _this select 3;
+_name = _array select 0;
+BTC_tk_PVEH = [_name,name player];
+publicVariable "BTC_tk_PVEH";
+hint format ["%1 has committed TK and has been punished by %2",_name, name player];
+player removeAction _id;
\ No newline at end of file
diff -r 993591e7a9ee -r 3d6f83ca9716 awia/=BTC=_TK_punishment/=BTC=_tk_init.sqf
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/awia/=BTC=_TK_punishment/=BTC=_tk_init.sqf	Fri Sep 27 15:20:23 2013 +0400
@@ -0,0 +1,123 @@
+/*
+	Created by =BTC= Giallustio
+	version 0.3
+	Visit us at: 
+	http://www.blacktemplars.altervista.org/
+	06/03/2012
+*/
+//Def
+BTC_tk_blackscreen_punishment = 2;
+BTC_tk_last_warning = 6;
+
+BTC_fnc_tk_PVEH =
+{
+	//0 - first aid - create // [0,east,pos]
+	//1 - first aid - delete
+	_array = _this select 1;
+	_name  = _array select 0;
+	_punisher = _array select 1;
+	hint format ["%1 has committed TK and has been punished by %2",_name,_punisher];
+	if (name player == _name) then {_spawn = [] spawn BTC_Teamkill;};
+};
+BTC_EH_killed =
+{
+	_body = _this select 0;
+	_killer = _this select 1;
+	_veh_body = vehicle _body;
+	_veh_killer = vehicle _killer;
+	_name = name _killer;
+	if (_name != name _body && BTC_vip find (name _killer) == -1) then
+	{
+		if (side _killer == BTC_side && _veh_body != _veh_killer) then
+		{
+			//BTC_tk_PVEH = [_name];publicVariable "BTC_tk_PVEH";player sidechat format ["%1 has committed TK",_name];
+			_killer spawn 
+			{
+				hint format ["%1 TK you! You can decide to punish him by action menu", name _this];
+				WaitUntil {Alive player};
+				_action = player addAction [("<t color=""#ED2744"">") + ("Punish " + name _this) + "</t>","=BTC=_TK_punishment\=BTC=_punish_action.sqf",[name _this], 8, true, true, "", "true"];
+				_timeout = time + 30;
+				WaitUntil {sleep 1; (_timeout < time)};
+				player removeAction _action;
+			};
+		};
+	};
+};
+BTC_Teamkill =
+{
+	player addrating 9999;
+	BTC_teamkiller = BTC_teamkiller + 1;
+	_uid = getPlayerUID player;
+	BTC_logic setVariable [_uid,BTC_teamkiller,true];
+	switch (true) do
+	{
+		case (BTC_teamkiller <= BTC_tk_blackscreen_punishment) : 
+		{
+			titleText ["STOP TEAMKILLING!","BLACK FADED"];
+			sleep 3;
+			titleText ["STOP TEAMKILLING!","PLAIN"];
+		};
+		case (BTC_teamkiller > BTC_tk_blackscreen_punishment && BTC_teamkiller <= BTC_tk_last_warning) : 
+		{
+			private ["_n","_array_tk","_msg"];
+			_msg = "";
+			disableUserInput true;
+			if (BTC_teamkiller == BTC_tk_last_warning) then
+			{
+				_msg = "STOP TEAMKILLING, LAST WARNING";
+			} else {_msg = "YOU HAVE BEEN PUNISHED FOR TEAMKILLING!";};
+			player setPos [0,0,0];
+			_n = 0;
+			while {_n < 60} do
+			{
+				titleText [_msg,"BLACK FADED"];
+				_n = _n + 1;
+				sleep 1;
+			};
+			disableUserInput false;
+			player setDamage 1;
+		};
+		case (BTC_teamkiller > BTC_tk_last_warning) : 
+		{
+
+			disableUserInput true;
+			titleText ["YOU HAVE BEEN PUNISHED FOR TEAMKILLING!\nYOU ARE NOT WELCOME ANYMORE","BLACK FADED"];
+			player setPos [0,0,0];
+			while {true} do
+			{
+				titleText ["YOU VE BEEN PUNISHED FOR TEAMKILLING!\nYOU ARE NOT WELCOME ANYMORE","BLACK FADED"];
+				sleep 1;
+			};
+			disableUserInput false;
+		};
+	};
+};
+if (isServer) then
+{
+	BTC_tk_PVEH = [];publicVariable "BTC_tk_PVEH";
+};
+if (!isDedicated) then
+{
+	[] spawn
+	{
+		private ["_uid","_array_tk"];
+		waitUntil {!isNull player};
+		waitUntil {player == player};
+		player addEventHandler ["Killed", BTC_EH_killed];
+		"BTC_tk_PVEH" addPublicVariableEventHandler BTC_fnc_tk_PVEH;
+		player addrating 9999;
+		BTC_side = side player;
+		BTC_vip = [];
+		_uid = getPlayerUID player;
+		if (isNil {BTC_logic getVariable _uid}) then
+		{
+			BTC_logic setVariable [_uid,0,true];
+			BTC_teamkiller = 0;
+		}
+		else
+		{
+			BTC_teamkiller = BTC_logic getVariable _uid;
+			if (BTC_teamkiller > BTC_tk_last_warning) then {_tk = [] spawn BTC_Teamkill;}
+		};
+	};
+};
\ No newline at end of file
diff -r 993591e7a9ee -r 3d6f83ca9716 awia/=BTC=_revive/=BTC=_respawn.h
--- a/awia/=BTC=_revive/=BTC=_respawn.h	Fri Sep 27 15:17:43 2013 +0400
+++ b/awia/=BTC=_revive/=BTC=_respawn.h	Fri Sep 27 15:20:23 2013 +0400
@@ -1,4 +1,4 @@
-class Button 
+﻿class Button 
 {
 	idc = -1;
 	type = 16;
diff -r 993591e7a9ee -r 3d6f83ca9716 awia/Thumbs.db
Binary file awia/Thumbs.db has changed
diff -r 993591e7a9ee -r 3d6f83ca9716 awia/description.ext
--- a/awia/description.ext	Fri Sep 27 15:17:43 2013 +0400
+++ b/awia/description.ext	Fri Sep 27 15:20:23 2013 +0400
@@ -384,10 +384,10 @@
 }; */
 
 /* Adds screen when loading. */
-author = Rarek [AW];
+author = Rarek [AW] (iBolit edit);
 onLoadName = "AW Invade & Annex";
 loadScreen = "awMissionLogo.jpg";
-OnLoadMission = "A custom-made domination-esque mission for ArmA 3, Ahoy World Invade &amp; Annex is lovingly crafted by Rarek. To follow / aid in the development of this mission, please sign up to www.AhoyWorld.co.uk and join us on TeamSpeak at ts.ahoyworld.co.uk:9987.";
+OnLoadMission = "Site: a3rc.com TS IP: a3rc.com";
 
 
 respawn = BASE;
diff -r 993591e7a9ee -r 3d6f83ca9716 awia/init.sqf
--- a/awia/init.sqf	Fri Sep 27 15:17:43 2013 +0400
+++ b/awia/init.sqf	Fri Sep 27 15:20:23 2013 +0400
@@ -25,11 +25,13 @@
 if (!isDedicated) then {waitUntil {!isNull player && isPlayer player};};
 
 #define WELCOME_MESSAGE	"Welcome to Ahoy World's Invade & Annex ~ALTIS~\n" +\
-						"by Rarek (Ahoy World)\n\n" +\
-						"To follow / aid in the development of this map, please register at\n" +\
-						"www.AhoyWorld.co.uk\n\n" +\
-						"...and feel free to join us on TeamSpeak at\n" +\
-						"ts.ahoyworld.co.uk"
+						"by Rarek (Ahoy World) edit by iBolit (A3RC)\n\n" +\
+						"www.a3rc.com\n\n" +\
+						"TeamSpeak:\n" +\
+						"a3rc.com\n\n" +\
+						"CHANGELOG:\n" +\
+						"v0.8: Removed HeyMaker (Artillery), Added UAV Operator and 2 UAV, Added admin debug"
+call compile preprocessFile "=BTC=_TK_punishment\=BTC=_tk_init.sqf";
 					
 					
 /* =============================================== */
diff -r 993591e7a9ee -r 3d6f83ca9716 awia/mission.sqm
--- a/awia/mission.sqm	Fri Sep 27 15:17:43 2013 +0400
+++ b/awia/mission.sqm	Fri Sep 27 15:20:23 2013 +0400
@@ -45,9 +45,9 @@
 	addOnsAuto[]=
 	{
 		"A3_Characters_F_BLUFOR",
+		"A3_Air_F_Gamma_UAV_02",
 		"A3_Characters_F_Gamma",
 		"a3_uav_f_characters_f_gamma",
-		"A3_Air_F_Gamma_UAV_02",
 		"A3_Misc_F_Helpers",
 		"A3_Soft_F_MRAP_01",
 		"A3_Signs_F",
@@ -77,13 +77,13 @@
 		"a3_soft_f_truck",
 		"A3_Structures_F_Mil_Shelters",
 		"A3_Soft_F_Car",
-		"A3_Armor_F_Slammer",
 		"a3_map_altis"
 	};
 	randomSeed=12935265;
 	class Intel
 	{
-		briefingName="co40 AW Invade & Annex -ALTIS- 2.51";
+		briefingName="co40 AW Invade & Annex -ALTIS- 2.51 (iBolit edit v0.8)";
+		overviewText="co40 AW Invade & Annex -ALTIS- 2.51 (iBolit edit v0.8)";
 		timeOfChanges=28800;
 		startWeather=0.099999994;
 		startWind=0.07;
@@ -105,7 +105,7 @@
 	};
 	class Groups
 	{
-		items=9;
+		items=12;
 		class Item0
 		{
 			side="WEST";
@@ -614,7 +614,7 @@
 					rank="LIEUTENANT";
 					skill=0.60000002;
 					init="(group this) setGroupId [""TV""]; ";
-					description="Vortex Pilot (TS3: ts.ahoyworld.co.uk)";
+					description="Vortex Pilot (TS3: a3rc.com)";
 				};
 				class Item1
 				{
@@ -626,19 +626,19 @@
 					player="PLAYER COMMANDER";
 					rank="SERGEANT";
 					skill=0.60000002;
-					description="Vortex Pilot (TS3: ts.ahoyworld.co.uk)";
+					description="Vortex Pilot (TS3: a3rc.com";
 				};
 				class Item2
 				{
 					position[]={14778.345,17.91,16799.854};
-					azimut=-144.52132;
+					azimut=-144.521;
 					id=38;
 					side="WEST";
 					vehicle="B_Helipilot_F";
 					player="PLAY CDG";
 					rank="CORPORAL";
 					skill=0.60000002;
-					description="Vortex Pilot (TS3: ts.ahoyworld.co.uk)";
+					description="Vortex Pilot (TS3: a3rc.com";
 				};
 			};
 		};
@@ -650,22 +650,95 @@
 				items=1;
 				class Item0
 				{
-					position[]={14735.546,18.278162,16934.488};
-					azimut=-2.346014;
+					position[]={15103.581,17.91,17253.639};
+					azimut=-225.13699;
 					id=39;
 					side="WEST";
+					vehicle="B_UAV_02_F";
+					isUAV=1;
+					leader=1;
+					skill=0.60000002;
+				};
+			};
+		};
+		class Item7
+		{
+			side="LOGIC";
+			class Vehicles
+			{
+				items=1;
+				class Item0
+				{
+					position[]={14766.3,-0.022443339,15405.381};
+					id=40;
+					side="LOGIC";
+					vehicle="Logic";
+					leader=1;
+					skill=0.60000002;
+					text="BTC_logic";
+				};
+			};
+		};
+		class Item8
+		{
+			side="WEST";
+			class Vehicles
+			{
+				items=1;
+				class Item0
+				{
+					position[]={15135.488,17.811337,17281.465};
+					azimut=-165.278;
+					id=41;
+					side="WEST";
 					vehicle="B_soldier_UAV_F";
 					player="PLAY CDG";
 					leader=1;
-					rank="CORPORAL";
 					skill=0.60000002;
-					text="mortarMan";
-					init="(group this) setGroupId [""ARTY - HayMaker""];";
-					description="Gunner - HayMaker";
+					init="(group this) setGroupId [""UAV""];";
+					description="UAV Operator";
 				};
 			};
 		};
-		class Item7
+		class Item9
+		{
+			side="WEST";
+			class Vehicles
+			{
+				items=1;
+				class Item0
+				{
+					position[]={15223.062,17.91,16797.338};
+					azimut=-494.116;
+					id=42;
+					side="WEST";
+					vehicle="B_UAV_02_F";
+					isUAV=1;
+					leader=1;
+					skill=0.60000002;
+				};
+			};
+		};
+		class Item10
+		{
+			side="WEST";
+			class Vehicles
+			{
+				items=1;
+				class Item0
+				{
+					position[]={15133.795,17.91,16853.264};
+					azimut=-139.918;
+					id=43;
+					side="WEST";
+					vehicle="B_UAV_02_CAS_F";
+					isUAV=1;
+					leader=1;
+					skill=0.60000002;
+				};
+			};
+		};
+		class Item11
 		{
 			side="WEST";
 			class Vehicles
@@ -675,44 +748,25 @@
 				{
 					position[]={15084.81,17.91,17233.25};
 					azimut=-225.13699;
-					id=40;
+					id=44;
 					side="WEST";
-					vehicle="B_UAV_02_F";
+					vehicle="B_UAV_02_CAS_F";
 					isUAV=1;
 					leader=1;
 					skill=0.60000002;
 				};
 			};
 		};
-		class Item8
-		{
-			side="WEST";
-			class Vehicles
-			{
-				items=1;
-				class Item0
-				{
-					position[]={15103.581,17.91,17253.639};
-					azimut=-225.13699;
-					id=41;
-					side="WEST";
-					vehicle="B_UAV_02_F";
-					isUAV=1;
-					leader=1;
-					skill=0.60000002;
-				};
-			};
-		};
 	};
 	class Vehicles
 	{
-		items=317;
+		items=315;
 		class Item0
 		{
 			position[]={15246.208,-0.0082027912,14204.392};
 			azimut=-146.1685;
 			offsetY=1.9;
-			id=42;
+			id=45;
 			side="EMPTY";
 			vehicle="UserTexture1m_F";
 			skill=0.60000002;
@@ -723,7 +777,7 @@
 			position[]={15220.312,0.10148622,14221.295};
 			azimut=-146.16901;
 			offsetY=1.9;
-			id=43;
+			id=46;
 			side="EMPTY";
 			vehicle="UserTexture1m_F";
 			skill=0.60000002;
@@ -733,7 +787,7 @@
 		{
 			position[]={14625.059,17.886612,16894.779};
 			azimut=-183.416;
-			id=44;
+			id=47;
 			side="EMPTY";
 			vehicle="B_MRAP_01_hmg_F";
 			leader=1;
@@ -746,7 +800,7 @@
 		{
 			position[]={14591.937,17.91,16837.352};
 			azimut=-404.03098;
-			id=45;
+			id=48;
 			side="EMPTY";
 			vehicle="RoadBarrier_F";
 			leader=1;
@@ -757,7 +811,7 @@
 		{
 			position[]={14486.246,17.346315,16838.135};
 			azimut=43.479668;
-			id=46;
+			id=49;
 			side="EMPTY";
 			vehicle="Land_Razorwire_F";
 			leader=1;
@@ -768,7 +822,7 @@
 		{
 			position[]={14481.1,17.109072,16843.182};
 			azimut=47.817493;
-			id=47;
+			id=50;
 			side="EMPTY";
 			vehicle="Land_Razorwire_F";
 			leader=1;
@@ -779,7 +833,7 @@
 		{
 			position[]={14582.063,17.91,16743.215};
 			azimut=-42.754299;
-			id=48;
+			id=51;
 			side="EMPTY";
 			vehicle="Land_BarGate_F";
 			skill=0.60000002;
@@ -789,7 +843,7 @@
 		{
 			position[]={14591.72,17.796196,16899.396};
 			azimut=-633.52002;
-			id=49;
+			id=52;
 			side="EMPTY";
 			vehicle="B_MRAP_01_F";
 			leader=1;
@@ -800,7 +854,7 @@
 		{
 			position[]={14484.418,17.061357,16852.268};
 			azimut=-44.857025;
-			id=50;
+			id=53;
 			side="EMPTY";
 			vehicle="Land_BagBunker_Small_F";
 			skill=0.60000002;
@@ -809,7 +863,7 @@
 		{
 			position[]={14590.053,17.91,16835.49};
 			azimut=-44.408772;
-			id=51;
+			id=54;
 			side="EMPTY";
 			vehicle="Land_LampShabby_F";
 			skill=0.60000002;
@@ -818,7 +872,7 @@
 		{
 			position[]={14595.538,17.91,16840.896};
 			azimut=-404.03098;
-			id=52;
+			id=55;
 			side="EMPTY";
 			vehicle="RoadBarrier_F";
 			leader=1;
@@ -829,7 +883,7 @@
 		{
 			position[]={14499.587,17.559294,16795.965};
 			azimut=-766.383;
-			id=53;
+			id=56;
 			side="EMPTY";
 			vehicle="B_Truck_01_covered_F";
 			leader=1;
@@ -841,7 +895,7 @@
 		{
 			position[]={14646.919,17.91,16742.289};
 			azimut=-47.452446;
-			id=54;
+			id=57;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -853,7 +907,7 @@
 			position[]={14655.039,17.91,16804.941};
 			azimut=313.08655;
 			offsetY=0.5;
-			id=55;
+			id=58;
 			side="EMPTY";
 			vehicle="Land_CampingTable_F";
 			leader=1;
@@ -865,7 +919,7 @@
 			position[]={14653.701,17.91,16803.529};
 			azimut=313.08655;
 			offsetY=0.5;
-			id=56;
+			id=59;
 			side="EMPTY";
 			vehicle="Land_CampingTable_F";
 			leader=1;
@@ -877,7 +931,7 @@
 			position[]={14652.367,17.91,16802.137};
 			azimut=313.08655;
 			offsetY=0.5;
-			id=57;
+			id=60;
 			side="EMPTY";
 			vehicle="Land_CampingTable_F";
 			leader=1;
@@ -889,7 +943,7 @@
 			position[]={14651.065,17.91,16800.723};
 			azimut=313.08655;
 			offsetY=0.5;
-			id=58;
+			id=61;
 			side="EMPTY";
 			vehicle="Land_CampingTable_F";
 			leader=1;
@@ -901,7 +955,7 @@
 			position[]={14654.346,17.91,16806.061};
 			azimut=338.08643;
 			offsetY=0.5;
-			id=59;
+			id=62;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -913,7 +967,7 @@
 			position[]={14653.809,17.91,16805.242};
 			azimut=313.08655;
 			offsetY=0.5;
-			id=60;
+			id=63;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -925,7 +979,7 @@
 			position[]={14653.139,17.91,16804.529};
 			azimut=313.08655;
 			offsetY=0.5;
-			id=61;
+			id=64;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -937,7 +991,7 @@
 			position[]={14652.414,17.91,16803.783};
 			azimut=343.08633;
 			offsetY=0.5;
-			id=62;
+			id=65;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -949,7 +1003,7 @@
 			position[]={14651.451,17.91,16803.359};
 			azimut=313.08655;
 			offsetY=0.5;
-			id=63;
+			id=66;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -961,7 +1015,7 @@
 			position[]={14650.946,17.91,16802.26};
 			azimut=313.08655;
 			offsetY=0.5;
-			id=64;
+			id=67;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -973,7 +1027,7 @@
 			position[]={14650.25,17.91,16801.438};
 			azimut=313.08655;
 			offsetY=0.5;
-			id=65;
+			id=68;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -985,7 +1039,7 @@
 			position[]={14652.025,17.91,16799.719};
 			azimut=133.08614;
 			offsetY=0.5;
-			id=66;
+			id=69;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -997,7 +1051,7 @@
 			position[]={14652.739,17.91,16800.422};
 			azimut=133.08614;
 			offsetY=0.5;
-			id=67;
+			id=70;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1009,7 +1063,7 @@
 			position[]={14653.394,17.91,16801.23};
 			azimut=193.08611;
 			offsetY=0.5;
-			id=68;
+			id=71;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1021,7 +1075,7 @@
 			position[]={14654.231,17.91,16802.061};
 			azimut=133.08614;
 			offsetY=0.5;
-			id=69;
+			id=72;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1033,7 +1087,7 @@
 			position[]={14654.828,17.91,16803.021};
 			azimut=133.08614;
 			offsetY=0.5;
-			id=70;
+			id=73;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1045,7 +1099,7 @@
 			position[]={14655.614,17.91,16803.738};
 			azimut=133.08614;
 			offsetY=0.5;
-			id=71;
+			id=74;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1057,7 +1111,7 @@
 			position[]={14656.31,17.91,16804.447};
 			azimut=133.08614;
 			offsetY=0.5;
-			id=72;
+			id=75;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1068,7 +1122,7 @@
 		{
 			position[]={14625.704,17.91,16774.506};
 			azimut=-45.969635;
-			id=73;
+			id=76;
 			side="EMPTY";
 			vehicle="Land_CampingTable_F";
 			leader=1;
@@ -1079,7 +1133,7 @@
 		{
 			position[]={14624.344,17.91,16773.123};
 			azimut=-45.969635;
-			id=74;
+			id=77;
 			side="EMPTY";
 			vehicle="Land_CampingTable_F";
 			leader=1;
@@ -1090,7 +1144,7 @@
 		{
 			position[]={14623.006,17.91,16771.711};
 			azimut=-45.969635;
-			id=75;
+			id=78;
 			side="EMPTY";
 			vehicle="Land_CampingTable_F";
 			leader=1;
@@ -1101,7 +1155,7 @@
 		{
 			position[]={14621.625,17.91,16770.49};
 			azimut=-45.969635;
-			id=76;
+			id=79;
 			side="EMPTY";
 			vehicle="Land_CampingTable_F";
 			leader=1;
@@ -1112,7 +1166,7 @@
 		{
 			position[]={14624.978,17.91,16775.695};
 			azimut=-20.969578;
-			id=77;
+			id=80;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1123,7 +1177,7 @@
 		{
 			position[]={14624.437,17.91,16774.857};
 			azimut=-45.969635;
-			id=78;
+			id=81;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1134,7 +1188,7 @@
 		{
 			position[]={14623.725,17.91,16774.221};
 			azimut=-45.969635;
-			id=79;
+			id=82;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1145,7 +1199,7 @@
 		{
 			position[]={14623.014,17.91,16773.496};
 			azimut=-15.969584;
-			id=80;
+			id=83;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1156,7 +1210,7 @@
 		{
 			position[]={14622.046,17.91,16773.127};
 			azimut=-45.969635;
-			id=81;
+			id=84;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1167,7 +1221,7 @@
 		{
 			position[]={14621.52,17.91,16772.023};
 			azimut=-45.969635;
-			id=82;
+			id=85;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1178,7 +1232,7 @@
 		{
 			position[]={14620.805,17.91,16771.229};
 			azimut=-45.969635;
-			id=83;
+			id=86;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1189,7 +1243,7 @@
 		{
 			position[]={14622.555,17.91,16769.473};
 			azimut=-225.96954;
-			id=84;
+			id=87;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1200,7 +1254,7 @@
 		{
 			position[]={14623.246,17.91,16770.184};
 			azimut=-225.96954;
-			id=85;
+			id=88;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1211,7 +1265,7 @@
 		{
 			position[]={14623.953,17.91,16770.959};
 			azimut=-165.96948;
-			id=86;
+			id=89;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1222,7 +1276,7 @@
 		{
 			position[]={14624.833,17.91,16771.623};
 			azimut=-225.96954;
-			id=87;
+			id=90;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1233,7 +1287,7 @@
 		{
 			position[]={14625.476,17.91,16772.592};
 			azimut=-225.96954;
-			id=88;
+			id=91;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1244,7 +1298,7 @@
 		{
 			position[]={14626.262,17.91,16773.314};
 			azimut=-185.96951;
-			id=89;
+			id=92;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1255,7 +1309,7 @@
 		{
 			position[]={14626.956,17.91,16774.004};
 			azimut=-225.96954;
-			id=90;
+			id=93;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1266,7 +1320,7 @@
 		{
 			position[]={14650.122,17.91,16745.033};
 			azimut=-226.22606;
-			id=91;
+			id=94;
 			side="EMPTY";
 			vehicle="Land_BarrelWater_F";
 			leader=1;
@@ -1276,7 +1330,7 @@
 		{
 			position[]={14649.662,17.91,16744.551};
 			azimut=-226.22606;
-			id=92;
+			id=95;
 			side="EMPTY";
 			vehicle="Land_BarrelWater_F";
 			leader=1;
@@ -1286,7 +1340,7 @@
 		{
 			position[]={14649.193,17.91,16744.037};
 			azimut=-226.22606;
-			id=93;
+			id=96;
 			side="EMPTY";
 			vehicle="Land_BarrelWater_F";
 			leader=1;
@@ -1296,7 +1350,7 @@
 		{
 			position[]={14648.73,17.91,16743.557};
 			azimut=-226.22606;
-			id=94;
+			id=97;
 			side="EMPTY";
 			vehicle="Land_BarrelWater_F";
 			leader=1;
@@ -1306,7 +1360,7 @@
 		{
 			position[]={14654.92,17.91,16750.047};
 			azimut=-226.22606;
-			id=95;
+			id=98;
 			side="EMPTY";
 			vehicle="Land_BarrelWater_F";
 			leader=1;
@@ -1316,7 +1370,7 @@
 		{
 			position[]={14654.439,17.91,16749.559};
 			azimut=-226.22606;
-			id=96;
+			id=99;
 			side="EMPTY";
 			vehicle="Land_BarrelWater_F";
 			leader=1;
@@ -1327,7 +1381,7 @@
 			position[]={14622.117,17.91,16799.785};
 			azimut=403.31409;
 			offsetY=0.5;
-			id=97;
+			id=100;
 			side="EMPTY";
 			vehicle="Land_CampingTable_F";
 			leader=1;
@@ -1339,7 +1393,7 @@
 			position[]={14623.242,17.91,16800.523};
 			azimut=428.31415;
 			offsetY=0.5;
-			id=98;
+			id=101;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1351,7 +1405,7 @@
 			position[]={14620.871,17.91,16799.25};
 			azimut=223.31377;
 			offsetY=0.5;
-			id=99;
+			id=102;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1363,7 +1417,7 @@
 			position[]={14621.577,17.91,16798.547};
 			azimut=223.31377;
 			offsetY=0.5;
-			id=100;
+			id=103;
 			side="EMPTY";
 			vehicle="Land_CampingChair_V2_F";
 			leader=1;
@@ -1374,7 +1428,7 @@
 		{
 			position[]={14644.636,17.91,16739.936};
 			azimut=-47.452446;
-			id=101;
+			id=104;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1385,7 +1439,7 @@
 		{
 			position[]={14642.441,17.91,16737.529};
 			azimut=-47.452446;
-			id=102;
+			id=105;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1396,7 +1450,7 @@
 		{
 			position[]={14640.165,17.91,16735.115};
 			azimut=-47.452446;
-			id=103;
+			id=106;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1407,7 +1461,7 @@
 		{
 			position[]={14663.491,17.91,16759.793};
 			azimut=-47.452446;
-			id=104;
+			id=107;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1418,7 +1472,7 @@
 		{
 			position[]={14661.118,17.91,16757.324};
 			azimut=-47.452446;
-			id=105;
+			id=108;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1429,7 +1483,7 @@
 		{
 			position[]={14658.904,17.91,16754.936};
 			azimut=-47.452446;
-			id=106;
+			id=109;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1440,7 +1494,7 @@
 		{
 			position[]={14656.65,17.91,16752.537};
 			azimut=-47.45269;
-			id=107;
+			id=110;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1451,7 +1505,7 @@
 		{
 			position[]={14576.233,17.91,16825.818};
 			azimut=-314.22589;
-			id=108;
+			id=111;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -1462,7 +1516,7 @@
 		{
 			position[]={14592.682,17.91,16842.553};
 			azimut=-312.23941;
-			id=109;
+			id=112;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -1473,7 +1527,7 @@
 		{
 			position[]={14574.063,17.91,16823.805};
 			azimut=-314.22589;
-			id=110;
+			id=113;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -1484,7 +1538,7 @@
 		{
 			position[]={14578.454,17.91,16827.879};
 			azimut=-314.22589;
-			id=111;
+			id=114;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -1495,7 +1549,7 @@
 		{
 			position[]={14571.553,17.91,16821.023};
 			azimut=-314.22589;
-			id=112;
+			id=115;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -1506,7 +1560,7 @@
 		{
 			position[]={14590.205,17.91,16839.975};
 			azimut=-312.23941;
-			id=113;
+			id=116;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -1517,7 +1571,7 @@
 		{
 			position[]={14587.743,17.91,16837.43};
 			azimut=-312.23941;
-			id=114;
+			id=117;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -1528,7 +1582,7 @@
 		{
 			position[]={14585.262,17.91,16835.057};
 			azimut=-312.23941;
-			id=115;
+			id=118;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -1539,7 +1593,7 @@
 		{
 			position[]={14580.88,17.91,16830.385};
 			azimut=-314.22589;
-			id=116;
+			id=119;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -1550,7 +1604,7 @@
 		{
 			position[]={14583.222,17.91,16832.828};
 			azimut=-312.23941;
-			id=117;
+			id=120;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -1561,7 +1615,7 @@
 		{
 			position[]={14550.817,17.91,16795.621};
 			azimut=-403.16733;
-			id=118;
+			id=121;
 			side="EMPTY";
 			vehicle="RoadBarrier_F";
 			leader=1;
@@ -1572,7 +1626,7 @@
 		{
 			position[]={14546.969,17.91,16791.314};
 			azimut=-403.54526;
-			id=119;
+			id=122;
 			side="EMPTY";
 			vehicle="RoadBarrier_F";
 			leader=1;
@@ -1583,7 +1637,7 @@
 		{
 			position[]={14467.002,17.211493,16821.148};
 			azimut=-136.07416;
-			id=120;
+			id=123;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1594,7 +1648,7 @@
 		{
 			position[]={14464.447,17.116217,16823.627};
 			azimut=-136.07416;
-			id=121;
+			id=124;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1605,7 +1659,7 @@
 		{
 			position[]={14461.885,17.031456,16826.119};
 			azimut=-136.07416;
-			id=122;
+			id=125;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1616,7 +1670,7 @@
 		{
 			position[]={14465.739,17.158373,16822.465};
 			azimut=-141.07416;
-			id=123;
+			id=126;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -1627,7 +1681,7 @@
 		{
 			position[]={14463.183,17.07408,16824.891};
 			azimut=-141.07416;
-			id=124;
+			id=127;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -1638,7 +1692,7 @@
 		{
 			position[]={14460.62,16.989046,16827.412};
 			azimut=-141.07416;
-			id=125;
+			id=128;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -1649,7 +1703,7 @@
 		{
 			position[]={14454.359,16.919922,16832.771};
 			azimut=-141.07416;
-			id=126;
+			id=129;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -1660,7 +1714,7 @@
 		{
 			position[]={14468.152,17.251129,16819.9};
 			azimut=-141.07416;
-			id=127;
+			id=130;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -1671,7 +1725,7 @@
 		{
 			position[]={14436.686,16.766674,16815.189};
 			azimut=-49.806026;
-			id=128;
+			id=131;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -1682,7 +1736,7 @@
 		{
 			position[]={14437.838,16.78211,16816.477};
 			azimut=-44.806038;
-			id=129;
+			id=132;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1693,7 +1747,7 @@
 		{
 			position[]={14440.373,16.815563,16818.959};
 			azimut=-44.806038;
-			id=130;
+			id=133;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1704,7 +1758,7 @@
 		{
 			position[]={14442.932,16.837696,16821.49};
 			azimut=-44.806038;
-			id=131;
+			id=134;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1715,7 +1769,7 @@
 		{
 			position[]={14445.494,16.860775,16824.037};
 			azimut=-44.806038;
-			id=132;
+			id=135;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1726,7 +1780,7 @@
 		{
 			position[]={14447.997,16.894278,16826.574};
 			azimut=-44.806038;
-			id=133;
+			id=136;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1737,7 +1791,7 @@
 		{
 			position[]={14450.577,16.903572,16829.055};
 			azimut=-44.806038;
-			id=134;
+			id=137;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1748,7 +1802,7 @@
 		{
 			position[]={14453.104,16.913223,16831.529};
 			azimut=-44.806038;
-			id=135;
+			id=138;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1759,7 +1813,7 @@
 		{
 			position[]={14439.183,16.799339,16817.717};
 			azimut=-49.806026;
-			id=136;
+			id=139;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -1770,7 +1824,7 @@
 		{
 			position[]={14441.657,16.827543,16820.227};
 			azimut=-49.806026;
-			id=137;
+			id=140;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -1781,7 +1835,7 @@
 		{
 			position[]={14444.247,16.847878,16822.721};
 			azimut=-49.806026;
-			id=138;
+			id=141;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -1792,7 +1846,7 @@
 		{
 			position[]={14446.758,16.877569,16825.287};
 			azimut=-49.806026;
-			id=139;
+			id=142;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -1803,7 +1857,7 @@
 		{
 			position[]={14449.321,16.900225,16827.799};
 			azimut=-49.806026;
-			id=140;
+			id=143;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -1814,7 +1868,7 @@
 		{
 			position[]={14451.862,16.907757,16830.229};
 			azimut=-49.806026;
-			id=141;
+			id=144;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -1825,7 +1879,7 @@
 		{
 			position[]={14430.514,16.68807,16809.072};
 			azimut=-44.806038;
-			id=142;
+			id=145;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1836,7 +1890,7 @@
 		{
 			position[]={14432.954,16.733477,16811.494};
 			azimut=-44.806038;
-			id=143;
+			id=146;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1847,7 +1901,7 @@
 		{
 			position[]={14435.403,16.756044,16813.957};
 			azimut=-44.806038;
-			id=144;
+			id=147;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -1858,7 +1912,7 @@
 		{
 			position[]={14431.778,16.71162,16810.33};
 			azimut=-49.806026;
-			id=145;
+			id=148;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -1869,7 +1923,7 @@
 		{
 			position[]={14434.255,16.746548,16812.693};
 			azimut=-49.806026;
-			id=146;
+			id=149;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -1880,7 +1934,7 @@
 		{
 			position[]={14455.09,16.923819,16831.377};
 			azimut=-135.17503;
-			id=147;
+			id=150;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -1891,7 +1945,7 @@
 		{
 			position[]={14456.439,16.931015,16831.014};
 			azimut=-135.17503;
-			id=148;
+			id=151;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -1902,7 +1956,7 @@
 		{
 			position[]={14458.993,16.959553,16828.4};
 			azimut=-135.17503;
-			id=149;
+			id=152;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -1913,7 +1967,7 @@
 		{
 			position[]={14458.923,16.968679,16827.383};
 			azimut=-135.17503;
-			id=150;
+			id=153;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -1925,7 +1979,7 @@
 			position[]={14649.408,17.91,16798.934};
 			azimut=223.62343;
 			offsetY=0.5;
-			id=151;
+			id=154;
 			side="EMPTY";
 			vehicle="MapBoard_altis_F";
 			skill=0.60000002;
@@ -1934,7 +1988,7 @@
 		{
 			position[]={14654.06,17.91,16749.09};
 			azimut=-226.22606;
-			id=152;
+			id=155;
 			side="EMPTY";
 			vehicle="Land_BarrelWater_F";
 			leader=1;
@@ -1944,7 +1998,7 @@
 		{
 			position[]={14627.215,17.91,16775.926};
 			azimut=42.853699;
-			id=153;
+			id=156;
 			side="EMPTY";
 			vehicle="MapBoard_altis_F";
 			skill=0.60000002;
@@ -1953,7 +2007,7 @@
 		{
 			position[]={14627.195,17.91,16855.439};
 			azimut=-855.91425;
-			id=154;
+			id=157;
 			side="EMPTY";
 			vehicle="B_Truck_01_covered_F";
 			leader=1;
@@ -1964,7 +2018,7 @@
 		{
 			position[]={14653.122,17.91,16842.367};
 			azimut=-71.026802;
-			id=155;
+			id=158;
 			side="EMPTY";
 			vehicle="Land_FirePlace_F";
 			skill=0.60000002;
@@ -1974,7 +2028,7 @@
 		{
 			position[]={14563.512,17.764841,16868.475};
 			azimut=224.229;
-			id=156;
+			id=159;
 			side="EMPTY";
 			vehicle="B_Quadbike_01_F";
 			skill=0.60000002;
@@ -1984,7 +2038,7 @@
 		{
 			position[]={14565.706,17.769596,16866.348};
 			azimut=224.229;
-			id=157;
+			id=160;
 			side="EMPTY";
 			vehicle="B_Quadbike_01_F";
 			skill=0.60000002;
@@ -1994,7 +2048,7 @@
 		{
 			position[]={14567.884,17.774977,16864.193};
 			azimut=224.229;
-			id=158;
+			id=161;
 			side="EMPTY";
 			vehicle="B_Quadbike_01_F";
 			skill=0.60000002;
@@ -2004,7 +2058,7 @@
 		{
 			position[]={14570.053,17.780842,16862.021};
 			azimut=224.229;
-			id=159;
+			id=162;
 			side="EMPTY";
 			vehicle="B_Quadbike_01_F";
 			skill=0.60000002;
@@ -2014,7 +2068,7 @@
 		{
 			position[]={14554.518,17.747532,16876.967};
 			azimut=224.229;
-			id=160;
+			id=163;
 			side="EMPTY";
 			vehicle="B_Quadbike_01_F";
 			skill=0.60000002;
@@ -2024,7 +2078,7 @@
 		{
 			position[]={14556.934,17.753822,16874.939};
 			azimut=224.229;
-			id=161;
+			id=164;
 			side="EMPTY";
 			vehicle="B_Quadbike_01_F";
 			skill=0.60000002;
@@ -2034,7 +2088,7 @@
 		{
 			position[]={14559.114,17.759989,16872.797};
 			azimut=224.229;
-			id=162;
+			id=165;
 			side="EMPTY";
 			vehicle="B_Quadbike_01_F";
 			skill=0.60000002;
@@ -2044,7 +2098,7 @@
 		{
 			position[]={14561.406,17.761837,16870.596};
 			azimut=224.229;
-			id=163;
+			id=166;
 			side="EMPTY";
 			vehicle="B_Quadbike_01_F";
 			skill=0.60000002;
@@ -2054,7 +2108,7 @@
 		{
 			position[]={14596.869,17.813852,16894.609};
 			azimut=85.932999;
-			id=164;
+			id=167;
 			side="EMPTY";
 			vehicle="B_MRAP_01_F";
 			leader=1;
@@ -2065,7 +2119,7 @@
 		{
 			position[]={14602.059,17.820608,16889.543};
 			azimut=446.70599;
-			id=165;
+			id=168;
 			side="EMPTY";
 			vehicle="B_MRAP_01_F";
 			leader=1;
@@ -2076,7 +2130,7 @@
 		{
 			position[]={14585.881,17.770943,16903.836};
 			azimut=86.581398;
-			id=166;
+			id=169;
 			side="EMPTY";
 			vehicle="B_MRAP_01_F";
 			leader=1;
@@ -2087,7 +2141,7 @@
 		{
 			position[]={14439.435,16.82687,16822.281};
 			azimut=-226.22098;
-			id=167;
+			id=170;
 			side="EMPTY";
 			vehicle="Land_LampShabby_F";
 			skill=0.60000002;
@@ -2096,7 +2150,7 @@
 		{
 			position[]={14635.17,17.91,16729.828};
 			azimut=-590.08429;
-			id=168;
+			id=171;
 			side="EMPTY";
 			vehicle="Sign_F";
 			skill=0.60000002;
@@ -2107,7 +2161,7 @@
 		{
 			position[]={14667.95,17.91,16764.594};
 			azimut=-226.37108;
-			id=169;
+			id=172;
 			side="EMPTY";
 			vehicle="Sign_F";
 			skill=0.60000002;
@@ -2118,7 +2172,7 @@
 		{
 			position[]={14651.584,17.91,16745.531};
 			azimut=-224.89137;
-			id=170;
+			id=173;
 			side="EMPTY";
 			vehicle="Sign_F";
 			skill=0.60000002;
@@ -2129,7 +2183,7 @@
 		{
 			position[]={14637.446,17.91,16732.246};
 			azimut=-135.48608;
-			id=171;
+			id=174;
 			side="EMPTY";
 			vehicle="Land_LampShabby_F";
 			skill=0.60000002;
@@ -2138,7 +2192,7 @@
 		{
 			position[]={14665.843,17.91,16762.52};
 			azimut=-319.92441;
-			id=172;
+			id=175;
 			side="EMPTY";
 			vehicle="Land_LampShabby_F";
 			skill=0.60000002;
@@ -2147,7 +2201,7 @@
 		{
 			position[]={15419.388,0.086374559,15801.756};
 			azimut=-189.00285;
-			id=173;
+			id=176;
 			side="EMPTY";
 			vehicle="B_Boat_Armed_01_minigun_F";
 			leader=1;
@@ -2159,7 +2213,7 @@
 		{
 			position[]={15429.615,0.19540089,15755.4};
 			azimut=-541.46075;
-			id=174;
+			id=177;
 			side="EMPTY";
 			vehicle="B_Boat_Armed_01_minigun_F";
 			leader=1;
@@ -2171,7 +2225,7 @@
 		{
 			position[]={15321.131,-0.017785773,15795.646};
 			azimut=-543.57721;
-			id=175;
+			id=178;
 			side="EMPTY";
 			vehicle="B_SDV_01_F";
 			leader=1;
@@ -2183,7 +2237,7 @@
 		{
 			position[]={15323.471,0.29443079,15758.658};
 			azimut=-543.57721;
-			id=176;
+			id=179;
 			side="EMPTY";
 			vehicle="B_SDV_01_F";
 			leader=1;
@@ -2195,7 +2249,7 @@
 		{
 			position[]={14621.185,17.91,16841.268};
 			azimut=-45.496899;
-			id=177;
+			id=180;
 			side="EMPTY";
 			vehicle="B_MRAP_01_F";
 			leader=1;
@@ -2206,7 +2260,7 @@
 		{
 			position[]={14616.969,17.91,16837.508};
 			azimut=314;
-			id=178;
+			id=181;
 			side="EMPTY";
 			vehicle="B_MRAP_01_F";
 			leader=1;
@@ -2217,7 +2271,7 @@
 		{
 			position[]={14613.201,17.91,16833.596};
 			azimut=-46.390999;
-			id=179;
+			id=182;
 			side="EMPTY";
 			vehicle="B_MRAP_01_F";
 			leader=1;
@@ -2228,7 +2282,7 @@
 		{
 			position[]={14652.822,17.91,16747.846};
 			azimut=-227.30901;
-			id=180;
+			id=183;
 			side="EMPTY";
 			vehicle="FlagPole_F";
 			leader=1;
@@ -2240,7 +2294,7 @@
 		{
 			position[]={14652.318,17.91,16747.27};
 			azimut=-228.58296;
-			id=181;
+			id=184;
 			side="EMPTY";
 			vehicle="FlagPole_F";
 			leader=1;
@@ -2252,7 +2306,7 @@
 		{
 			position[]={14653.437,17.91,16748.473};
 			azimut=-227.30916;
-			id=182;
+			id=185;
 			side="EMPTY";
 			vehicle="FlagPole_F";
 			leader=1;
@@ -2265,7 +2319,7 @@
 			position[]={15255.945,0.0019117892,14216.824};
 			azimut=-146.1685;
 			offsetY=1.9;
-			id=183;
+			id=186;
 			side="EMPTY";
 			vehicle="UserTexture1m_F";
 			skill=0.60000002;
@@ -2276,7 +2330,7 @@
 			position[]={15228.757,-0.027227059,14233.893};
 			azimut=-146.16901;
 			offsetY=1.9;
-			id=184;
+			id=187;
 			side="EMPTY";
 			vehicle="UserTexture1m_F";
 			skill=0.60000002;
@@ -2287,7 +2341,7 @@
 			position[]={15263.904,0.0094882995,14229.752};
 			azimut=-146.1685;
 			offsetY=1.9;
-			id=185;
+			id=188;
 			side="EMPTY";
 			vehicle="UserTexture1m_F";
 			skill=0.60000002;
@@ -2298,7 +2352,7 @@
 			position[]={15238.011,0.16027151,14246.659};
 			azimut=-146.16901;
 			offsetY=1.9;
-			id=186;
+			id=189;
 			side="EMPTY";
 			vehicle="UserTexture1m_F";
 			skill=0.60000002;
@@ -2308,7 +2362,7 @@
 		{
 			position[]={14741.315,18.407465,16939.627};
 			azimut=-226.28505;
-			id=187;
+			id=190;
 			side="EMPTY";
 			vehicle="Land_ClutterCutter_large_F";
 			skill=0.60000002;
@@ -2317,7 +2371,7 @@
 		{
 			position[]={14734.887,18.258425,16933.473};
 			azimut=-224.022;
-			id=188;
+			id=191;
 			side="EMPTY";
 			vehicle="Land_ClutterCutter_large_F";
 			skill=0.60000002;
@@ -2326,7 +2380,7 @@
 		{
 			position[]={14734.949,18.10815,16945.807};
 			azimut=-226.28505;
-			id=189;
+			id=192;
 			side="EMPTY";
 			vehicle="Land_ClutterCutter_large_F";
 			skill=0.60000002;
@@ -2335,7 +2389,7 @@
 		{
 			position[]={14728.706,18.225636,16939.732};
 			azimut=-226.28505;
-			id=190;
+			id=193;
 			side="EMPTY";
 			vehicle="Land_ClutterCutter_large_F";
 			skill=0.60000002;
@@ -2344,7 +2398,7 @@
 		{
 			position[]={14722.321,18.2313,16945.506};
 			azimut=-226.28505;
-			id=191;
+			id=194;
 			side="EMPTY";
 			vehicle="Land_ClutterCutter_large_F";
 			skill=0.60000002;
@@ -2353,7 +2407,7 @@
 		{
 			position[]={14728.397,17.961515,16951.699};
 			azimut=-226.28505;
-			id=192;
+			id=195;
 			side="EMPTY";
 			vehicle="Land_ClutterCutter_large_F";
 			skill=0.60000002;
@@ -2362,7 +2416,7 @@
 		{
 			position[]={14674.267,17.91,16807.014};
 			azimut=181.403;
-			id=193;
+			id=196;
 			side="EMPTY";
 			vehicle="B_supplyCrate_F";
 			skill=0.60000002;
@@ -2373,7 +2427,7 @@
 		{
 			position[]={14652.891,17.91,16751.133};
 			azimut=132.29657;
-			id=194;
+			id=197;
 			side="EMPTY";
 			vehicle="B_supplyCrate_F";
 			skill=0.60000002;
@@ -2384,7 +2438,7 @@
 		{
 			position[]={14629.055,17.91,16777.596};
 			azimut=43.115501;
-			id=195;
+			id=198;
 			side="EMPTY";
 			vehicle="B_supplyCrate_F";
 			skill=0.60000002;
@@ -2395,7 +2449,7 @@
 		{
 			position[]={14647.705,17.91,16797.311};
 			azimut=401.21671;
-			id=196;
+			id=199;
 			side="EMPTY";
 			vehicle="B_supplyCrate_F";
 			skill=0.60000002;
@@ -2406,7 +2460,7 @@
 		{
 			position[]={14609.051,17.91,16820.744};
 			azimut=136.44099;
-			id=197;
+			id=200;
 			side="EMPTY";
 			vehicle="B_supplyCrate_F";
 			skill=0.60000002;
@@ -2417,7 +2471,7 @@
 		{
 			position[]={14552.244,17.91,16801.705};
 			azimut=-314.22589;
-			id=198;
+			id=201;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2428,7 +2482,7 @@
 		{
 			position[]={14568.692,17.91,16818.439};
 			azimut=-312.23941;
-			id=199;
+			id=202;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2439,7 +2493,7 @@
 		{
 			position[]={14550.074,17.91,16799.691};
 			azimut=-314.22589;
-			id=200;
+			id=203;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2450,7 +2504,7 @@
 		{
 			position[]={14554.465,17.91,16803.766};
 			azimut=-314.22589;
-			id=201;
+			id=204;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2461,7 +2515,7 @@
 		{
 			position[]={14547.563,17.91,16796.91};
 			azimut=-314.22589;
-			id=202;
+			id=205;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2472,7 +2526,7 @@
 		{
 			position[]={14566.216,17.91,16815.861};
 			azimut=-312.23941;
-			id=203;
+			id=206;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2483,7 +2537,7 @@
 		{
 			position[]={14563.754,17.91,16813.316};
 			azimut=-312.23941;
-			id=204;
+			id=207;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2494,7 +2548,7 @@
 		{
 			position[]={14561.272,17.91,16810.943};
 			azimut=-312.23941;
-			id=205;
+			id=208;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2505,7 +2559,7 @@
 		{
 			position[]={14556.891,17.91,16806.271};
 			azimut=-314.22589;
-			id=206;
+			id=209;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2516,7 +2570,7 @@
 		{
 			position[]={14559.232,17.91,16808.715};
 			azimut=-312.23941;
-			id=207;
+			id=210;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2527,7 +2581,7 @@
 		{
 			position[]={14568.974,17.91,16834.73};
 			azimut=-314.22589;
-			id=208;
+			id=211;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2538,7 +2592,7 @@
 		{
 			position[]={14585.422,17.91,16851.465};
 			azimut=-312.23941;
-			id=209;
+			id=212;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2549,7 +2603,7 @@
 		{
 			position[]={14566.804,17.91,16832.717};
 			azimut=-314.22589;
-			id=210;
+			id=213;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2560,7 +2614,7 @@
 		{
 			position[]={14571.194,17.91,16836.791};
 			azimut=-314.22589;
-			id=211;
+			id=214;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2571,7 +2625,7 @@
 		{
 			position[]={14564.293,17.91,16829.936};
 			azimut=-314.22589;
-			id=212;
+			id=215;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2582,7 +2636,7 @@
 		{
 			position[]={14582.945,17.91,16848.887};
 			azimut=-312.23941;
-			id=213;
+			id=216;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2593,7 +2647,7 @@
 		{
 			position[]={14580.483,17.91,16846.342};
 			azimut=-312.23941;
-			id=214;
+			id=217;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2604,7 +2658,7 @@
 		{
 			position[]={14578.002,17.91,16843.969};
 			azimut=-312.23941;
-			id=215;
+			id=218;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2615,7 +2669,7 @@
 		{
 			position[]={14573.62,17.91,16839.297};
 			azimut=-314.22589;
-			id=216;
+			id=219;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2626,7 +2680,7 @@
 		{
 			position[]={14575.962,17.91,16841.74};
 			azimut=-312.23941;
-			id=217;
+			id=220;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2637,7 +2691,7 @@
 		{
 			position[]={14544.984,17.91,16810.617};
 			azimut=-314.22589;
-			id=218;
+			id=221;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2648,7 +2702,7 @@
 		{
 			position[]={14561.433,17.91,16827.352};
 			azimut=-312.23941;
-			id=219;
+			id=222;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2659,7 +2713,7 @@
 		{
 			position[]={14542.814,17.91,16808.604};
 			azimut=-314.22589;
-			id=220;
+			id=223;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2670,7 +2724,7 @@
 		{
 			position[]={14547.205,17.91,16812.678};
 			azimut=-314.22589;
-			id=221;
+			id=224;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2681,7 +2735,7 @@
 		{
 			position[]={14540.304,17.91,16805.822};
 			azimut=-314.22589;
-			id=222;
+			id=225;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2692,7 +2746,7 @@
 		{
 			position[]={14558.956,17.91,16824.773};
 			azimut=-312.23941;
-			id=223;
+			id=226;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2703,7 +2757,7 @@
 		{
 			position[]={14556.494,17.91,16822.229};
 			azimut=-312.23941;
-			id=224;
+			id=227;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2714,7 +2768,7 @@
 		{
 			position[]={14554.013,17.91,16819.855};
 			azimut=-312.23941;
-			id=225;
+			id=228;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2725,7 +2779,7 @@
 		{
 			position[]={14549.631,17.91,16815.184};
 			azimut=-314.22589;
-			id=226;
+			id=229;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2736,7 +2790,7 @@
 		{
 			position[]={14551.973,17.91,16817.627};
 			azimut=-312.23941;
-			id=227;
+			id=230;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2747,7 +2801,7 @@
 		{
 			position[]={14477.047,16.988878,16857.186};
 			azimut=-135.17503;
-			id=228;
+			id=231;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2758,7 +2812,7 @@
 		{
 			position[]={14475.974,16.976412,16855.73};
 			azimut=-135.17503;
-			id=229;
+			id=232;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2769,7 +2823,7 @@
 		{
 			position[]={14474.544,16.957687,16851.676};
 			azimut=-135.17503;
-			id=230;
+			id=233;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2780,7 +2834,7 @@
 		{
 			position[]={14475.443,16.964067,16849.777};
 			azimut=-135.17503;
-			id=231;
+			id=234;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2791,7 +2845,7 @@
 		{
 			position[]={14477.021,16.989014,16847.967};
 			azimut=-135.17503;
-			id=232;
+			id=235;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2802,7 +2856,7 @@
 		{
 			position[]={14473.003,16.966417,16856.902};
 			azimut=-45.418385;
-			id=233;
+			id=236;
 			side="EMPTY";
 			vehicle="ArrowDesk_R_F";
 			skill=0.60000002;
@@ -2812,7 +2866,7 @@
 		{
 			position[]={14475.091,16.952318,16845.547};
 			azimut=-133.91325;
-			id=234;
+			id=237;
 			side="EMPTY";
 			vehicle="ArrowDesk_L_F";
 			skill=0.60000002;
@@ -2822,7 +2876,7 @@
 		{
 			position[]={14478.337,17.000298,16858.533};
 			azimut=-135.17503;
-			id=235;
+			id=238;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2833,7 +2887,7 @@
 		{
 			position[]={14475.008,16.963255,16853.695};
 			azimut=-135.17503;
-			id=236;
+			id=239;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2844,7 +2898,7 @@
 		{
 			position[]={14478.529,17.013252,16846.545};
 			azimut=-135.17503;
-			id=237;
+			id=240;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2855,7 +2909,7 @@
 		{
 			position[]={14480.101,17.015219,16860.301};
 			azimut=-135.17503;
-			id=238;
+			id=241;
 			side="EMPTY";
 			vehicle="RoadCone_F";
 			leader=1;
@@ -2866,7 +2920,7 @@
 		{
 			position[]={14484.137,17.060452,16862.697};
 			azimut=-50.677933;
-			id=239;
+			id=242;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -2877,7 +2931,7 @@
 		{
 			position[]={14485.274,17.07937,16864.016};
 			azimut=-45.677937;
-			id=240;
+			id=243;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -2888,7 +2942,7 @@
 		{
 			position[]={14487.759,17.107815,16866.543};
 			azimut=-45.677937;
-			id=241;
+			id=244;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -2899,7 +2953,7 @@
 		{
 			position[]={14490.275,17.135103,16869.115};
 			azimut=-45.677937;
-			id=242;
+			id=245;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -2910,7 +2964,7 @@
 		{
 			position[]={14492.813,17.165613,16871.701};
 			azimut=-45.677937;
-			id=243;
+			id=246;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -2921,7 +2975,7 @@
 		{
 			position[]={14495.272,17.16498,16874.27};
 			azimut=-45.677937;
-			id=244;
+			id=247;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -2932,7 +2986,7 @@
 		{
 			position[]={14486.604,17.09535,16865.275};
 			azimut=-50.677933;
-			id=245;
+			id=248;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -2943,7 +2997,7 @@
 		{
 			position[]={14489.046,17.122362,16867.814};
 			azimut=-50.677933;
-			id=246;
+			id=249;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -2954,7 +3008,7 @@
 		{
 			position[]={14491.583,17.150707,16870.359};
 			azimut=-50.677933;
-			id=247;
+			id=250;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -2965,7 +3019,7 @@
 		{
 			position[]={14494.065,17.167171,16872.953};
 			azimut=-50.677933;
-			id=248;
+			id=251;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -2976,7 +3030,7 @@
 		{
 			position[]={14483.035,17.043064,16861.658};
 			azimut=-45.677937;
-			id=249;
+			id=252;
 			side="EMPTY";
 			vehicle="Land_CncBarrier_F";
 			leader=1;
@@ -2987,7 +3041,7 @@
 		{
 			position[]={14481.987,17.029196,16860.65};
 			azimut=-50.677933;
-			id=250;
+			id=253;
 			side="EMPTY";
 			vehicle="Land_HBarrier_1_F";
 			leader=1;
@@ -2998,7 +3052,7 @@
 		{
 			position[]={14481.182,17.0404,16864.795};
 			azimut=-226.22098;
-			id=251;
+			id=254;
 			side="EMPTY";
 			vehicle="Land_LampShabby_F";
 			skill=0.60000002;
@@ -3007,7 +3061,7 @@
 		{
 			position[]={14577.866,17.91,16823.314};
 			azimut=-44.408772;
-			id=252;
+			id=255;
 			side="EMPTY";
 			vehicle="Land_LampShabby_F";
 			skill=0.60000002;
@@ -3016,7 +3070,7 @@
 		{
 			position[]={14565.681,17.91,16810.816};
 			azimut=-44.408772;
-			id=253;
+			id=256;
 			side="EMPTY";
 			vehicle="Land_LampShabby_F";
 			skill=0.60000002;
@@ -3025,7 +3079,7 @@
 		{
 			position[]={14552.657,17.91,16797.547};
 			azimut=-44.408772;
-			id=254;
+			id=257;
 			side="EMPTY";
 			vehicle="Land_LampShabby_F";
 			skill=0.60000002;
@@ -3034,7 +3088,7 @@
 		{
 			position[]={14573.018,17.778667,16862.035};
 			azimut=-44.408772;
-			id=255;
+			id=258;
 			side="EMPTY";
 			vehicle="Land_LampShabby_F";
 			skill=0.60000002;
@@ -3043,7 +3097,7 @@
 		{
 			position[]={14629.763,17.962448,16895.371};
 			azimut=-94.209183;
-			id=256;
+			id=259;
 			side="EMPTY";
 			vehicle="Land_LampShabby_F";
 			skill=0.60000002;
@@ -3052,7 +3106,7 @@
 		{
 			position[]={14631.758,17.91,16853.113};
 			azimut=-44.408772;
-			id=257;
+			id=260;
 			side="EMPTY";
 			vehicle="Land_LampShabby_F";
 			skill=0.60000002;
@@ -3061,7 +3115,7 @@
 		{
 			position[]={14559.158,17.878672,16847.953};
 			azimut=-44.408772;
-			id=258;
+			id=261;
 			side="EMPTY";
 			vehicle="Land_LampShabby_F";
 			skill=0.60000002;
@@ -3070,7 +3124,7 @@
 		{
 			position[]={14545.297,17.85998,16834.102};
 			azimut=-44.408772;
-			id=259;
+			id=262;
 			side="EMPTY";
 			vehicle="Land_LampShabby_F";
 			skill=0.60000002;
@@ -3079,7 +3133,7 @@
 		{
 			position[]={14531.434,17.736198,16820.248};
 			azimut=-44.408772;
-			id=260;
+			id=263;
 			side="EMPTY";
 			vehicle="Land_LampShabby_F";
 			skill=0.60000002;
@@ -3088,7 +3142,7 @@
 		{
 			position[]={14503.197,17.566309,16799.121};
 			azimut=-135.83162;
-			id=261;
+			id=264;
 			side="EMPTY";
 			vehicle="Land_LampShabby_F";
 			skill=0.60000002;
@@ -3098,7 +3152,7 @@
 			position[]={14487.357,17.458817,16806.771};
 			azimut=-43.691898;
 			offsetY=0.5;
-			id=262;
+			id=265;
 			side="EMPTY";
 			vehicle="B_APC_Tracked_01_rcws_F";
 			skill=0.60000002;
@@ -3109,7 +3163,7 @@
 			position[]={14510.665,17.530415,16833.676};
 			azimut=-135.91;
 			offsetY=0.5;
-			id=263;
+			id=266;
 			side="EMPTY";
 			vehicle="B_APC_Wheeled_01_cannon_F";
 			lock="UNLOCKED";
@@ -3120,7 +3174,7 @@
 		{
 			position[]={21103.303,21.915039,7251.917};
 			azimut=-77.731949;
-			id=264;
+			id=267;
 			side="EMPTY";
 			vehicle="Land_HelipadSquare_F";
 			leader=1;
@@ -3131,7 +3185,7 @@
 		{
 			position[]={9133.1631,17.016048,21474.561};
 			azimut=-128.23111;
-			id=265;
+			id=268;
 			side="EMPTY";
 			vehicle="Land_HelipadSquare_F";
 			leader=1;
@@ -3142,7 +3196,7 @@
 		{
 			position[]={26997.906,21.390556,24722.895};
 			azimut=37.597027;
-			id=266;
+			id=269;
 			side="EMPTY";
 			vehicle="Land_HelipadSquare_F";
 			leader=1;
@@ -3153,7 +3207,7 @@
 		{
 			position[]={14543.833,17.804211,16852.99};
 			azimut=-855.914;
-			id=267;
+			id=270;
 			side="EMPTY";
 			vehicle="B_Truck_01_medical_F";
 			leader=1;
@@ -3164,7 +3218,7 @@
 		{
 			position[]={14549.559,17.818825,16847.334};
 			azimut=-855.914;
-			id=268;
+			id=271;
 			side="EMPTY";
 			vehicle="B_Truck_01_Repair_F";
 			leader=1;
@@ -3175,7 +3229,7 @@
 		{
 			position[]={14534.437,17.673244,16838.156};
 			azimut=-855.914;
-			id=269;
+			id=272;
 			side="EMPTY";
 			vehicle="B_APC_Tracked_01_CRV_F";
 			leader=1;
@@ -3187,7 +3241,7 @@
 			position[]={14553.283,17.91,16770.371};
 			azimut=-42.754299;
 			offsetY=4.4000001;
-			id=270;
+			id=273;
 			side="EMPTY";
 			vehicle="Land_BarGate_F";
 			skill=0.60000002;
@@ -3199,7 +3253,7 @@
 			position[]={14547.554,17.91,16765.15};
 			azimut=-42.754299;
 			special="NONE";
-			id=271;
+			id=274;
 			side="EMPTY";
 			vehicle="Land_BagFence_Long_F";
 			skill=0.60000002;
@@ -3210,7 +3264,7 @@
 			position[]={14544.989,17.91,16762.834};
 			azimut=-42.754299;
 			special="NONE";
-			id=272;
+			id=275;
 			side="EMPTY";
 			vehicle="Land_BagFence_Long_F";
 			skill=0.60000002;
@@ -3220,7 +3274,7 @@
 			position[]={14575.33,17.91,16737.164};
 			azimut=-42.754299;
 			special="NONE";
-			id=273;
+			id=276;
 			side="EMPTY";
 			vehicle="Land_BagFence_Long_F";
 			skill=0.60000002;
@@ -3231,7 +3285,7 @@
 			position[]={14572.766,17.91,16734.848};
 			azimut=-42.754299;
 			special="NONE";
-			id=274;
+			id=277;
 			side="EMPTY";
 			vehicle="Land_BagFence_Long_F";
 			skill=0.60000002;
@@ -3241,7 +3295,7 @@
 			position[]={14553.488,17.91,16771.078};
 			azimut=-42.754299;
 			offsetY=4.4000001;
-			id=275;
+			id=278;
 			side="EMPTY";
 			vehicle="Land_HelipadEmpty_F";
 			skill=0.60000002;
@@ -3253,7 +3307,7 @@
 			azimut=-42.754299;
 			offsetY=4.4000001;
 			special="NONE";
-			id=276;
+			id=279;
 			side="EMPTY";
 			vehicle="Land_HelipadEmpty_F";
 			skill=0.60000002;
@@ -3264,7 +3318,7 @@
 		{
 			position[]={14817.593,17.91,16809.578};
 			azimut=223.106;
-			id=277;
+			id=280;
 			side="EMPTY";
 			vehicle="I_Heli_Transport_02_F";
 			leader=1;
@@ -3275,7 +3329,7 @@
 		{
 			position[]={14800.71,17.91,16825.91};
 			azimut=222.084;
-			id=278;
+			id=281;
 			side="EMPTY";
 			vehicle="I_Heli_Transport_02_F";
 			leader=1;
@@ -3287,7 +3341,7 @@
 		{
 			position[]={14774.068,17.91,16780.072};
 			azimut=-317.909;
-			id=279;
+			id=282;
 			side="EMPTY";
 			vehicle="B_Heli_Light_01_F";
 			lock="UNLOCKED";
@@ -3300,7 +3354,7 @@
 		{
 			position[]={14752.884,17.91,16799.609};
 			azimut=-314.62601;
-			id=280;
+			id=283;
 			side="EMPTY";
 			vehicle="B_Heli_Light_01_armed_F";
 			skill=0.60000002;
@@ -3310,7 +3364,7 @@
 		{
 			position[]={14781.602,17.91,16844.072};
 			azimut=222.8;
-			id=281;
+			id=284;
 			side="EMPTY";
 			vehicle="B_Heli_Transport_01_camo_F";
 			leader=1;
@@ -3322,7 +3376,7 @@
 		{
 			position[]={14539.557,17.747995,16857.633};
 			azimut=-855.914;
-			id=282;
+			id=285;
 			side="EMPTY";
 			vehicle="B_Truck_01_fuel_F";
 			leader=1;
@@ -3333,7 +3387,7 @@
 		{
 			position[]={14534.705,17.652327,16861.994};
 			azimut=-855.914;
-			id=283;
+			id=286;
 			side="EMPTY";
 			vehicle="B_Truck_01_ammo_F";
 			leader=1;
@@ -3344,7 +3398,7 @@
 		{
 			position[]={14919.066,17.91,16741.52};
 			azimut=-856.30682;
-			id=284;
+			id=287;
 			side="EMPTY";
 			vehicle="B_G_Van_01_fuel_F";
 			leader=1;
@@ -3355,7 +3409,7 @@
 		{
 			position[]={14610.097,17.91,16634.703};
 			azimut=-42.521599;
-			id=285;
+			id=288;
 			side="EMPTY";
 			vehicle="Land_HelipadSquare_F";
 			skill=0.60000002;
@@ -3365,7 +3419,7 @@
 			position[]={14564.834,17.91,16822.203};
 			azimut=-313.93381;
 			special="NONE";
-			id=286;
+			id=289;
 			side="EMPTY";
 			vehicle="Land_HelipadSquare_F";
 			leader=1;
@@ -3375,7 +3429,7 @@
 		class Item245
 		{
 			position[]={14674.078,17.91,16810.508};
-			id=287;
+			id=290;
 			side="EMPTY";
 			vehicle="CamoNet_BLUFOR_big_F";
 			skill=0.60000002;
@@ -3385,7 +3439,7 @@
 		{
 			position[]={14672.293,17.91,16729.355};
 			azimut=-48.5611;
-			id=288;
+			id=291;
 			side="EMPTY";
 			vehicle="Land_HelipadCivil_F";
 			skill=0.60000002;
@@ -3393,7 +3447,7 @@
 		class Item247
 		{
 			position[]={14753.101,17.91,16800.084};
-			id=289;
+			id=292;
 			side="EMPTY";
 			vehicle="Land_HelipadCivil_F";
 			skill=0.60000002;
@@ -3401,7 +3455,7 @@
 		class Item248
 		{
 			position[]={14774.132,17.91,16780.609};
-			id=290;
+			id=293;
 			side="EMPTY";
 			vehicle="Land_HelipadCivil_F";
 			skill=0.60000002;
@@ -3409,7 +3463,7 @@
 		class Item249
 		{
 			position[]={14816.696,17.91,16808.719};
-			id=291;
+			id=294;
 			side="EMPTY";
 			vehicle="Land_HelipadCivil_F";
 			skill=0.60000002;
@@ -3417,7 +3471,7 @@
 		class Item250
 		{
 			position[]={14799.902,17.91,16824.895};
-			id=292;
+			id=295;
 			side="EMPTY";
 			vehicle="Land_HelipadCivil_F";
 			skill=0.60000002;
@@ -3425,7 +3479,7 @@
 		class Item251
 		{
 			position[]={14780.745,17.91,16842.822};
-			id=293;
+			id=296;
 			side="EMPTY";
 			vehicle="Land_HelipadCivil_F";
 			skill=0.60000002;
@@ -3434,7 +3488,7 @@
 		{
 			position[]={14560.737,17.91,16806.076};
 			azimut=-403.16733;
-			id=294;
+			id=297;
 			side="EMPTY";
 			vehicle="RoadBarrier_F";
 			leader=1;
@@ -3445,7 +3499,7 @@
 		{
 			position[]={14556.889,17.91,16801.77};
 			azimut=-403.54526;
-			id=295;
+			id=298;
 			side="EMPTY";
 			vehicle="RoadBarrier_F";
 			leader=1;
@@ -3456,7 +3510,7 @@
 		{
 			position[]={14573.492,17.91,16819.146};
 			azimut=-403.16733;
-			id=296;
+			id=299;
 			side="EMPTY";
 			vehicle="RoadBarrier_F";
 			leader=1;
@@ -3467,7 +3521,7 @@
 		{
 			position[]={14569.644,17.91,16814.84};
 			azimut=-403.54526;
-			id=297;
+			id=300;
 			side="EMPTY";
 			vehicle="RoadBarrier_F";
 			leader=1;
@@ -3478,7 +3532,7 @@
 		{
 			position[]={14585.92,17.91,16831.563};
 			azimut=-403.16733;
-			id=298;
+			id=301;
 			side="EMPTY";
 			vehicle="RoadBarrier_F";
 			leader=1;
@@ -3489,7 +3543,7 @@
 		{
 			position[]={14582.071,17.91,16827.256};
 			azimut=-403.54526;
-			id=299;
+			id=302;
 			side="EMPTY";
 			vehicle="RoadBarrier_F";
 			leader=1;
@@ -3500,7 +3554,7 @@
 		{
 			position[]={14643.679,17.91,16700.525};
 			azimut=-48.5611;
-			id=300;
+			id=303;
 			side="EMPTY";
 			vehicle="Land_HelipadCivil_F";
 			skill=0.60000002;
@@ -3509,7 +3563,7 @@
 		{
 			position[]={14698.529,17.91,16757.512};
 			azimut=-48.5611;
-			id=301;
+			id=304;
 			side="EMPTY";
 			vehicle="Land_HelipadCivil_F";
 			skill=0.60000002;
@@ -3519,7 +3573,7 @@
 			position[]={16028.77,13.364062,17035.402};
 			azimut=119.21732;
 			special="NONE";
-			id=302;
+			id=305;
 			side="EMPTY";
 			vehicle="C_Hatchback_01_sport_F";
 			leader=1;
@@ -3530,7 +3584,7 @@
 		{
 			position[]={14752.923,18.450117,16933.49};
 			azimut=193.21123;
-			id=303;
+			id=306;
 			side="EMPTY";
 			vehicle="FlagPole_F";
 			leader=1;
@@ -3544,7 +3598,7 @@
 			azimut=43.842747;
 			offsetY=-3;
 			special="NONE";
-			id=304;
+			id=307;
 			side="EMPTY";
 			vehicle="Land_LampHalogen_F";
 			leader=1;
@@ -3555,7 +3609,7 @@
 		{
 			position[]={14743.402,18.167763,16951.559};
 			azimut=-136.36899;
-			id=305;
+			id=308;
 			side="EMPTY";
 			vehicle="Land_LampShabby_F";
 			skill=0.60000002;
@@ -3564,7 +3618,7 @@
 		{
 			position[]={14744.98,18.28661,16925.738};
 			azimut=135.94501;
-			id=306;
+			id=309;
 			side="EMPTY";
 			vehicle="B_supplyCrate_F";
 			skill=0.60000002;
@@ -3573,31 +3627,9 @@
 		};
 		class Item265
 		{
-			position[]={14732.452,18.270878,16942.578};
-			azimut=-45.265984;
-			id=307;
-			side="EMPTY";
-			vehicle="B_MBT_01_arty_F";
-			skill=0.60000002;
-			fuel=0.31193987;
-			init="this removeMagazineTurret [""6Rnd_155mm_Mo_smoke"",[0]]; this removeMagazineTurret [""6Rnd_155mm_Mo_mine"",[0]]; this removeMagazineTurret [""6Rnd_155mm_Mo_AT_mine"",[0]]; this allowdamage false;";
-		};
-		class Item266
-		{
-			position[]={14738.223,18.305025,16925.703};
-			azimut=313.78799;
-			id=308;
-			side="EMPTY";
-			vehicle="B_Truck_01_ammo_F";
-			skill=0.60000002;
-			text="rearm2_1_1";
-			init="this allowdamage false;";
-		};
-		class Item267
-		{
 			position[]={14713.213,17.91,16943.852};
 			azimut=44.530399;
-			id=309;
+			id=310;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3605,12 +3637,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item268
+		class Item266
 		{
 			position[]={14713.24,17.91,16943.809};
 			azimut=44.530399;
 			offsetY=2.2;
-			id=310;
+			id=311;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3618,12 +3650,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item269
+		class Item267
 		{
 			position[]={14713.217,17.91,16943.824};
 			azimut=44.530399;
 			offsetY=4.4000001;
-			id=311;
+			id=312;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3631,11 +3663,11 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item270
+		class Item268
 		{
 			position[]={14719.316,17.967569,16937.871};
 			azimut=44.530399;
-			id=312;
+			id=313;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3643,12 +3675,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item271
+		class Item269
 		{
 			position[]={14719.344,17.967922,16937.828};
 			azimut=44.530399;
 			offsetY=2.2;
-			id=313;
+			id=314;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3656,12 +3688,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item272
+		class Item270
 		{
 			position[]={14719.32,17.967617,16937.844};
 			azimut=44.530399;
 			offsetY=4.4000001;
-			id=314;
+			id=315;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3669,11 +3701,11 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item273
+		class Item271
 		{
 			position[]={14725.381,18.00732,16931.895};
 			azimut=44.530399;
-			id=315;
+			id=316;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3681,12 +3713,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item274
+		class Item272
 		{
 			position[]={14725.408,18.007133,16931.852};
 			azimut=44.530399;
 			offsetY=2.2;
-			id=316;
+			id=317;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3694,12 +3726,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item275
+		class Item273
 		{
 			position[]={14725.385,18.007057,16931.867};
 			azimut=44.530399;
 			offsetY=4.4000001;
-			id=317;
+			id=318;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3707,11 +3739,11 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item276
+		class Item274
 		{
 			position[]={14731.484,18.032417,16925.914};
 			azimut=44.530399;
-			id=318;
+			id=319;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3719,12 +3751,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item277
+		class Item275
 		{
 			position[]={14731.512,18.032661,16925.871};
 			azimut=44.530399;
 			offsetY=2.2;
-			id=319;
+			id=320;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3732,12 +3764,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item278
+		class Item276
 		{
 			position[]={14731.488,18.032158,16925.887};
 			azimut=44.530399;
 			offsetY=4.4000001;
-			id=320;
+			id=321;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3745,11 +3777,11 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item279
+		class Item277
 		{
 			position[]={14737.588,18.128311,16919.934};
 			azimut=44.530399;
-			id=321;
+			id=322;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3757,12 +3789,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item280
+		class Item278
 		{
 			position[]={14737.615,18.126917,16919.891};
 			azimut=44.530399;
 			offsetY=2.2;
-			id=322;
+			id=323;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3770,12 +3802,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item281
+		class Item279
 		{
 			position[]={14737.592,18.12726,16919.906};
 			azimut=44.530399;
 			offsetY=4.4000001;
-			id=323;
+			id=324;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3783,11 +3815,11 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item282
+		class Item280
 		{
 			position[]={14712.112,17.91,16949.043};
 			azimut=-45.500591;
-			id=324;
+			id=325;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3795,12 +3827,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item283
+		class Item281
 		{
 			position[]={14712.165,17.91,16949.084};
 			azimut=-45.500591;
 			offsetY=2.2;
-			id=325;
+			id=326;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3808,12 +3840,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item284
+		class Item282
 		{
 			position[]={14712.145,17.91,16949.055};
 			azimut=-45.500591;
 			offsetY=4.4000001;
-			id=326;
+			id=327;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3821,11 +3853,11 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item285
+		class Item283
 		{
 			position[]={14718.094,17.9863,16955.158};
 			azimut=-45.500591;
-			id=327;
+			id=328;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3833,12 +3865,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item286
+		class Item284
 		{
 			position[]={14718.138,17.984512,16955.182};
 			azimut=-45.500591;
 			offsetY=2.2;
-			id=328;
+			id=329;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3846,12 +3878,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item287
+		class Item285
 		{
 			position[]={14718.12,17.98572,16955.162};
 			azimut=-45.500591;
 			offsetY=4.4000001;
-			id=329;
+			id=330;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3859,11 +3891,11 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item288
+		class Item286
 		{
 			position[]={14724.062,17.62636,16961.227};
 			azimut=-45.500591;
-			id=330;
+			id=331;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3871,12 +3903,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item289
+		class Item287
 		{
 			position[]={14724.113,17.625013,16961.248};
 			azimut=-45.500591;
 			offsetY=2.2;
-			id=331;
+			id=332;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3884,12 +3916,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item290
+		class Item288
 		{
 			position[]={14724.086,17.626129,16961.227};
 			azimut=-45.500591;
 			offsetY=4.4000001;
-			id=332;
+			id=333;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3897,11 +3929,11 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item291
+		class Item289
 		{
 			position[]={14728.377,17.491404,16961.391};
 			azimut=44.530399;
-			id=333;
+			id=334;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3909,12 +3941,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item292
+		class Item290
 		{
 			position[]={14728.404,17.493511,16961.348};
 			azimut=44.530399;
 			offsetY=2.2;
-			id=334;
+			id=335;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3922,12 +3954,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item293
+		class Item291
 		{
 			position[]={14728.381,17.493464,16961.363};
 			azimut=44.530399;
 			offsetY=4.4000001;
-			id=335;
+			id=336;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3935,11 +3967,11 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item294
+		class Item292
 		{
 			position[]={14734.48,17.839939,16955.41};
 			azimut=44.530399;
-			id=336;
+			id=337;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3947,12 +3979,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item295
+		class Item293
 		{
 			position[]={14734.508,17.840729,16955.367};
 			azimut=44.530399;
 			offsetY=2.2;
-			id=337;
+			id=338;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3960,12 +3992,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item296
+		class Item294
 		{
 			position[]={14734.484,17.84013,16955.383};
 			azimut=44.530399;
 			offsetY=4.4000001;
-			id=338;
+			id=339;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3973,11 +4005,11 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item297
+		class Item295
 		{
 			position[]={14740.545,18.078533,16949.434};
 			azimut=44.530399;
-			id=339;
+			id=340;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3985,12 +4017,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item298
+		class Item296
 		{
 			position[]={14740.572,18.081352,16949.391};
 			azimut=44.530399;
 			offsetY=2.2;
-			id=340;
+			id=341;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -3998,12 +4030,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item299
+		class Item297
 		{
 			position[]={14740.549,18.079782,16949.406};
 			azimut=44.530399;
 			offsetY=4.4000001;
-			id=341;
+			id=342;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -4011,11 +4043,11 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item300
+		class Item298
 		{
 			position[]={14746.648,18.574392,16943.453};
 			azimut=44.530399;
-			id=342;
+			id=343;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -4023,12 +4055,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item301
+		class Item299
 		{
 			position[]={14746.676,18.577286,16943.41};
 			azimut=44.530399;
 			offsetY=2.2;
-			id=343;
+			id=344;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -4036,12 +4068,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item302
+		class Item300
 		{
 			position[]={14746.652,18.575693,16943.426};
 			azimut=44.530399;
 			offsetY=4.4000001;
-			id=344;
+			id=345;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -4049,11 +4081,11 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item303
+		class Item301
 		{
 			position[]={14754.428,18.461533,16935.758};
 			azimut=44.530399;
-			id=345;
+			id=346;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -4061,12 +4093,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item304
+		class Item302
 		{
 			position[]={14754.455,18.458158,16935.715};
 			azimut=44.530399;
 			offsetY=2.2;
-			id=346;
+			id=347;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -4074,12 +4106,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item305
+		class Item303
 		{
 			position[]={14754.432,18.460043,16935.73};
 			azimut=44.530399;
 			offsetY=4.4000001;
-			id=347;
+			id=348;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -4087,11 +4119,11 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item306
+		class Item304
 		{
 			position[]={14754.561,18.293715,16931.344};
 			azimut=135.15442;
-			id=348;
+			id=349;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -4099,12 +4131,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item307
+		class Item305
 		{
 			position[]={14754.514,18.29417,16931.309};
 			azimut=135.15442;
 			offsetY=2.2;
-			id=349;
+			id=350;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -4112,12 +4144,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item308
+		class Item306
 		{
 			position[]={14754.533,18.294397,16931.334};
 			azimut=135.15442;
 			offsetY=4.4000001;
-			id=350;
+			id=351;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -4125,11 +4157,11 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item309
+		class Item307
 		{
 			position[]={14748.514,18.241522,16925.307};
 			azimut=135.15442;
-			id=351;
+			id=352;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -4137,12 +4169,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item310
+		class Item308
 		{
 			position[]={14748.475,18.241577,16925.273};
 			azimut=135.15442;
 			offsetY=2.2;
-			id=352;
+			id=353;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -4150,12 +4182,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item311
+		class Item309
 		{
 			position[]={14748.481,18.241774,16925.297};
 			azimut=135.15442;
 			offsetY=4.4000001;
-			id=353;
+			id=354;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -4163,11 +4195,11 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item312
+		class Item310
 		{
 			position[]={14742.461,18.153412,16919.262};
 			azimut=135.15442;
-			id=354;
+			id=355;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -4175,12 +4207,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item313
+		class Item311
 		{
 			position[]={14742.421,18.152031,16919.238};
 			azimut=135.15442;
 			offsetY=2.2;
-			id=355;
+			id=356;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -4188,12 +4220,12 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item314
+		class Item312
 		{
 			position[]={14742.432,18.152954,16919.258};
 			azimut=135.15442;
 			offsetY=4.4000001;
-			id=356;
+			id=357;
 			side="EMPTY";
 			vehicle="Land_HBarrierBig_F";
 			leader=1;
@@ -4201,22 +4233,22 @@
 			init="this allowdamage false;";
 			description="Gunner - HayMaker";
 		};
-		class Item315
+		class Item313
 		{
 			position[]={14604.555,17.766577,16917.893};
 			azimut=134.00665;
-			id=357;
+			id=358;
 			side="EMPTY";
 			vehicle="B_MRAP_01_F";
 			leader=1;
 			skill=0.60000002;
 			init="veh = [this, 60, 300, 0, FALSE, FALSE, ""nul = [this,""""C9"""",0.5] execVM """"mark_unit.sqf"""";""] execVM ""vehicle.sqf"";";
 		};
-		class Item316
+		class Item314
 		{
 			position[]={14614.433,17.833858,16908.227};
 			azimut=133.65759;
-			id=358;
+			id=359;
 			side="EMPTY";
 			vehicle="B_MRAP_01_F";
 			leader=1;
@@ -4227,7 +4259,7 @@
 	};
 	class Markers
 	{
-		items=64;
+		items=68;
 		class Item0
 		{
 			position[]={-1391.4594,-0.01276912,580.14929};
@@ -4679,14 +4711,14 @@
 		};
 		class Item61
 		{
-			position[]={14745.76,18.611528,16935.693};
+			position[]={14430.094,25.815588,6186.4717};
 			name="milhq";
 			type="Empty";
 			angle=94.516502;
 		};
 		class Item62
 		{
-			position[]={14742.118,18.415548,16932.584};
+			position[]={14427.36,26.098398,6187.1455};
 			name="MortarHQ";
 			type="Empty";
 		};
@@ -4703,6 +4735,38 @@
 			angle=176.68582;
 			drawBorder=1;
 		};
+		class Item64
+		{
+			position[]={12940.211,-72.139503,10845.061};
+			name="RTS";
+			text="TS: a3rc.com";
+			type="hd_flag";
+			colorName="ColorBlue";
+		};
+		class Item65
+		{
+			position[]={15168.386,17.882488,16976.199};
+			name="ts";
+			text="Teamspeak: a3rc.com";
+			type="hd_flag";
+			colorName="ColorBlue";
+		};
+		class Item66
+		{
+			position[]={15155.098,17.578232,17388.043};
+			name="sss";
+			text="КВ 123.4";
+			type="waypoint";
+			colorName="ColorRed";
+		};
+		class Item67
+		{
+			position[]={15161.629,17.91,17198.611};
+			name="sss_1";
+			text="ДВ 50";
+			type="waypoint";
+			colorName="ColorRed";
+		};
 	};
 	class Sensors
 	{
diff -r 993591e7a9ee -r 3d6f83ca9716 awia/rsc/Thumbs.db
Binary file awia/rsc/Thumbs.db has changed
